nextflow.config

```diff
@@ -32,15 +32,15 @@ params {
     fragment_bin_offset             = 0.0
     fragment_mass_tolerance         = 0.01
     number_mods                     = 3
-    fixed_mods                      = ' '
+    fixed_mods                      = ''
     variable_mods                   = 'Oxidation (M)'
     num_hits                        = 1
     use_x_ions                      = false
     use_z_ions                      = false
     use_a_ions                      = false
     use_c_ions                      = false
     use_NL_ions                     = false
-    remove_precursor_peak           = false
+    remove_precursor_peak           = 'no'
```

Comment:
I would suggest for keeping backwards compatibility for usrs to keep this as a boolean, and then 'translate to 'yes/no' in your modules.conf

Reply:
Good point!